src/wallet/wallet.cpp

```diff
@@ -2005,6 +2004,9 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
     {
         LOCK2(cs_main, cs_wallet);
         {
+            vector<COutput> vAvailableCoins;
```

Comment:
nit: please use std::vector for consistency and because we're striving to remove `using namespace std` at some point